src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -256,21 +269,25 @@ export class AccessibilityPolicyForm extends React.Component {
           />
           <TextArea
             name="message"
-            label="Message"
+            label={<WrappedMessage message={messages.label_message} />} // TODO: Can asInput be expanded to take an element for label?
             id="message"
             onChange={this.handleMessageChange}
             value={this.state.submitterMessage}
             themes={['danger']}
             validator={message => this.validateMessage(message)}
             inputRef={(ref) => { this.messageInputRef = ref; }}
           />
-          <Button
-            buttonType="primary"
-            label="Submit"
-            aria-label="Submit Accessibility Feedback Form"
-            onClick={() => { this.onSubmitClick(); }}
-            inputRef={(ref) => { this.submitButtonRef = ref; }}
-          />
+          <WrappedMessage message={messages.label_submit_aria}>
+            { displayText =>
+              (<Button
+                buttonType="primary"
+                label={<WrappedMessage message={messages.common.label_submit} />}
+                aria-label={displayText}
+                onClick={() => { this.onSubmitClick(); }}
+                inputRef={(ref) => { this.submitButtonRef = ref; }}
+              />)
+            }
+          </WrappedMessage>
```

Comment:
Here's how this looks once run, note that there are 2 different `lang` spans, these could be set to different languages (say, if we had translations available for one but not the other)

<img width="762" alt="screen shot 2018-01-29 at 11 09 31 am" src="https://user-images.githubusercontent.com/7373924/35520431-044d21f0-04e5-11e8-866b-4e8829ba4737.png">